changelog/README.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+* Added new page [Loyalty Program](../operations/loyaltyprogram.md) under [Operations](../operations/README.md).
```

Comment:
Couple of things:
- The new page is loyaltyprograms.md with an 's' and the name is 'Loyalty Programs' (plural), but it's without the 's' in the Changelog
- If a new page is added, SUMMARY.md needs updating as well, this is the index of all the pages used to create the navigation in GitBook